packages/fxa-payments-server/src/components/ProductValueProposition/index.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { MockLoader } from '../../../.storybook/components/MockLoader';
+import { ProductValueProposition } from './index';
+
+function init() {
+  storiesOf('components/ProductValueProposition', module)
+  .add('default', () => (
+    <MockLoader>
+      <ProductValueProposition plan={mockPlan} />
+    </MockLoader>
+  ))
```

Comment:
The default one gives me this, is that expected?

![image](https://user-images.githubusercontent.com/44390/58832537-c4c19680-8614-11e9-8bd5-eee696301c55.png)

Reply:
Yeah, this is just demo content right now - namely a [JSON.stringify() of the plan data](https://github.com/mozilla/fxa/blob/master/packages/fxa-payments-server/src/components/ProductValueProposition/index.tsx#L14).